src/main/scala/org/broadinstitute/monster/encode/extraction/EncodeClient.scala

```diff
@@ -0,0 +1,70 @@
+package org.broadinstitute.monster.encode.extraction
+
+import java.io.IOException
+import java.time.Duration
+
+import okhttp3.{Call, Callback, OkHttpClient, Request, Response}
+import org.slf4j.LoggerFactory
+
+import scala.concurrent.{Future, Promise}
+
+/** Interface for clients that hits ENCODE API. */
+trait EncodeClient extends Serializable {
+  def get(entity: EncodeEntity, params: List[(String, String)]): Future[String]
```

Comment:
Can we get away with being more specific?
```suggestion
  def get(entity: EncodeEntity, params: List[(String, String)]): Future[Msg]
```